docs/contributing/reporting-bugs.md

```diff
@@ -0,0 +1,155 @@
+   repository](https://github.com/zulip)**. The most commonly used repositories
+   are:
+   - [zulip/zulip](https://github.com/zulip/zulip/issues) for issues with the
+     Zulip web app or server. A good default if you aren't sure which repository
+     to use.
+   - [zulip/zulip-mobile](https://github.com/zulip/zulip-mobile/issues) for
+     issues with the mobile apps.
+   - [zulip/zulip-desktop](https://github.com/zulip/zulip-desktop/issues) for
+     issues that are specific to the Zulip desktop app, and _do not_ occur in
+     the web app.
+   - [zulip/zulip-terminal](https://github.com/zulip/zulip-terminal/issues) for
+     issues with the terminal app.
+2. Do a **quick search** of the repository to see if your issue has already
+   been filed. If it has, you can add a comment if that seems helpful.
+3. If you are aware of a related discussion in the Zulip development community,
+   please **cross-link** between the issue and the discussion thread. [Link to a
+   specific
+   message](https://zulip.com/help/link-to-a-message-or-conversation#get-a-link-to-a-specific-message)
+   in the discussion thread, as message links will still work even if the topic is
+   renamed or resolved.
+
+To encourage prompt attention and discussion for a bug report you have filed,
+you can send a message in the Zulip development community with the key points
+from your report. Be sure to [link to the GitHub
+issue](https://zulip.com/development-community/#linking-to-github-issues-and-pull-requests).
+See the following section for advice on where and how to start the conversation.
+
+## Starting a conversation about a possible bug
+
+If you are not sure whether the issue you encountered is a bug, or how to
+reproduce it, we highly recommend reporting it in the [Zulip development
+community](https://zulip.com/development-community/). It's the best place to
+interactively discuss your problem.
+
+Steps and best practices for starting a conversation:
+
+1. [**Join** the Zulip development
+   community](https://zulip.com/development-community/) if you don't already
+   have an account.
+2. Pick an **appropriate stream** to report your issue:
+   - [#issues](https://chat.zulip.org/#narrow/stream/9-issues) for issues with
+     the Zulip web app or server. A good default if you aren't sure which stream
+     to use.
+   - [#mobile](https://chat.zulip.org/#narrow/stream/48-mobile) for issues with
+     the mobile apps.
+   - [#desktop](https://chat.zulip.org/#narrow/stream/16-desktop) for issues
+     that are specific to the Zulip desktop app, and _do not_ occur in the web
+     app.
+   - [#zulip-terminal](https://chat.zulip.org/#narrow/stream/206-zulip-terminal)
+     for issues with the terminal app.
+   - [#production
+     help](https://chat.zulip.org/#narrow/stream/31-production-help) for issues
+     related to self-hosting Zulip. See the [troubleshooting
+     guide](../production/troubleshooting.md) for additional details.
```

Comment:
I think #feedback might still be correct here if it's a suggestion and not a supporty issue; @alexmv thoughts?